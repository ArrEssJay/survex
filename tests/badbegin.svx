; pos=fail error=2
*begin
*end  foo
*begin		    foo
*fix 1 reference 0 0 0
*end   bar
*begin omitted
*end
