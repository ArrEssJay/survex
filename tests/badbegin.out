./badbegin.svx:2:10: Matching BEGIN command has no survey name
./badbegin.svx:5:11: Survey name doesn't match BEGIN
./badbegin.svx:7:5: warning: Survey name omitted from END

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
Done.
There were 1 warning(s) and 2 error(s) - no output files produced.
